src/content/blog/astro-220.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: "Astro 2.2"
+description: "CDN support for assets • image() schema helper API change • Much more!"
+publishDate: "April 5, 2023"
+authors:
+  - matthew
+coverImage: "/src/content/blog/_images/astro-220/blog-hero-2_2-release.jpg"
+socialImage: "/src/content/blog/_images/astro-220/og-image-2_2-release.jpg"
+lang: "en"
+---
+
+Astro 2.2 is out! Included in this release are:
+
+- **[CDN support for assets](#cdn-support)**: You can now deploy your assets to a CDN external to your site, and Astro will loads from there.
+- **[image() schema helper API change](#image-schema-api-change)**: (experimental) The image() helper has been moved to be inline with the schema definition.
+
+Additionally Astro 2.2 includes **8 patch changes** and was released alongside **8 integrations** with patch releases.
+
+Run `npm i astro@latest` to upgrade to the latest version.
+
+## CDN support
+
+In 2.2 you can now configure Astro to point to a CDN in order to load your assets. In your Astro config:
+
+```js
+import { defineConfig } from "astro/config"
+
+export default defineConfig({
+	build: {
+		assetsPrefix: "https://cdn.example.com",
+	},
+})
+```
+
+After you've run the build (`astro build`) your HTML (both in SSG and SSR modes) will include links that include the prefix for any assets; scripts and stylesheets.
```

Comment:
```suggestion
The links for any assets, scripts, and stylesheets after running `astro build` will be prefixed with the configured URL. This applies to both SSG and SSR modes.
```
This is the only sentence I really did switch up. You don't have to use this, so switch it up as necessary! But there were a few fixes to be made to the original, and I think we could get it a bit smoother by changing it around a bit.